### PR TITLE
Chun translation

### PR DESCRIPTION
**Pull request by aryue123321**
- only made the translation of oneProp and Main Page
- made a function to parse the translation to JS object in translation.js.
- better each pages contains all the words even some common words.
  eg. do not use common.reset -> better use oneProportoin.reset, oneMean.reset.

### CODE REVIEW

**Comment by MatthewMerrill** [comment]
What CORS is rejected?

File: buildReleases.py [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+          spanish [category] = dict()
+        english[category][subKey] = en
+        spanish [category][subKey] = es
+  f = open(ymlFile, 'w',encoding='utf-8')
+  # print(english)
+  f.write(yaml.dump({ "en": english, "es": spanish }, default_flow_style=False, allow_unicode=True))
+
+def buildWebRelease():
+  print("\n #####  Building site_en #####\n")
+  subprocess.run('jekyll build JEKYLL_ENV=es -d ./_site_en/ JEKYLL_ENV=es', shell=True)
```
Comment (MatthewMerrill) [inline buildReleases.py]: I think you have the commands for English/Spanish reversed?

Comment (MatthewMerrill) [reply buildReleases.py]: Oh nvm, but you can remove the JEKYLL_ENV=es here. I don't think it does anything.

Comment (aryue123321) [reply buildReleases.py]: fixed, TY

**Pull request merged**